app/src/main/java/com/work/restaurant/view/search/rank/SearchRankFragment.kt

```diff
@@ -31,7 +32,7 @@ class SearchRankFragment : BaseFragment(R.layout.search_rank_fragment), View.OnC
     AdapterDataListener.GetKakaoData {
 
     private lateinit var presenter: SearchRankPresenter
-    private lateinit var kakaoAdapter: KakaoRankAdapter
+    private lateinit var searchRankAdapter: SearchRankAdapter
```

Comment:
```suggestion
    private val searchRankAdapter lazy { SearchRankAdapter() }
```
var타입이 필요한 경우가 아니라면, lateinit으로 늦게 init할 필요가 없을 것 같아요.

Reply:
fb49a24 수정하였습니다.